File: python/py_workers/worker.py
from py_workers.config import CeleryConfig
from celery import Celery

celery_app = Celery(
    main=CeleryConfig.CELERY_APP_NAME,
    backend=CeleryConfig.CELERY_RESULT_BACKEND,
    broker=CeleryConfig.BROKER_URL,
    include=[
        "py_workers.tasks.example.arithmetic",
    ],
)

queue = "pyqueue"

# celery_app.conf.task_routes = {'py_workers.tasks.*': {'queue': 'pyqueue'}}
celery_app.conf.task_routes = {"py_workers.tasks.*": {"queue": queue}}

if __name__ == "__main__":
    celery_app.start()

File: node/js_workers/worker.js
import celery from "celery-node";

import celeryConfig from './config.js'

import {add} from './tasks/example/arithmetic.js'

const worker = celery.createWorker(
  celeryConfig.BROKER_URL,
  celeryConfig.CELERY_RESULT_BACKEND,
  celeryConfig.QUEUE
);

worker.register("js_workers.tasks.example.arithmetic.add", add);
worker.start();
